app.py

```diff
@@ -1,85 +1,64 @@
+import json
 from dotenv import load_dotenv, find_dotenv
+from openai import OpenAI
 from flask import Flask, jsonify, request
-from authlib.integrations.flask_oauth2 import ResourceProtector
-import guidance
-import os
-
 
 ENV_FILE = find_dotenv()
 if ENV_FILE:
     load_dotenv(ENV_FILE)
 
-guidance.llm = guidance.llms.OpenAI("text-davinci-003", api_key=os.environ.get('OPENAI_API_KEY'), caching=False)
-
+client = OpenAI()
 app = Flask(__name__)
 
 @app.get("/")
 def hello_world():
   return {"hello": "world"}
 
-@app.post("/api/guidance/generate/board")
+@app.post("/api/generate/board")
 def generate_board():
-  title = request.form.get('title')
-  description = request.form.get('description')
-  detail_level = request.form.get('detail_level')
-  point_system = request.form.get('point_system') or 'Fibonacci Sequence (0, 1, 2, 3, 5)'
-
-  # {{#select "sprint_num" logprobs='logprobs'}}3{{or}}4{{or}}5{{/select}}
-  # {{#if suggested_time}} I planned on spending {{suggested_time}} on this project. {{/if}} 
-  # {{#if sprint_suggestions}}Make sure to include: {{#each sprint_suggestions}}{{this.suggestion}} in {{this.sprint}},{{/each}}{{/if}} 
-  # {{#if suggestions.timeframe}}Break up into 4, 2-week sprint intervals{{/if}} 
-  sprint_name_query = guidance('''
-    I'm making a project titled "{{title}}" with the description "{{description}}". How many sprints should I have to get started?
-    {{#select "sprint_num"}}3{{or}}4{{or}}5{{/select}}
-
-    Give me names for each sprint (make a maximum of {{sprint_num}} sprints):
-    {{#geneach 'sprint_titles' }}
-    Sprint {{@index}}: {{gen "this" max_tokens=5}}
-    {{/geneach}}
-  ''')
+  title=request.form.get('title')
+  description=request.form.get('description')
+  detail_level=request.form.get('detail_level') or 'detailed'
+  story_point_type=request.form.get('story_point_type') or 'Fibonacci Sequence'
+  story_points=request.form.get('story_points') or '0, 1, 2, 3, 5'
```

Comment:
is this supposed to be the only options for story points?because the example has 8 listed

Reply:
It is _supposed_ to but I everything with OpenAI is a suggestion. I mean we could make a check like If not in story_points set to default but for MVP I'm fine with this